clang/test/Driver/clang-translation.c

```diff
@@ -1,3 +1,6 @@
+// XFAIL: xcselect
```

Comment:
I don't feel XFAIL is correct to use here in general. It is possible to have a different host SDK and the test will actually pass, right?

Reply:
In theory, yes. In practice, it looks to me like all these failing test cases[^1] have deployment targets that pre-date macOS 10.14, so it seems very likely that anyone who can enable `CLANG_USE_XCSELECT` will probably see these tests fail as soon as they do.

But, point taken. Do you have an alternative approach in mind?

I'm wondering if it's worth reviving the `CLANG_NO_XCSELECT` environment variable and then just setting them in these tests (and slowly getting rid of instances of `CLANG_NO_XCSELECT` as we find better fixes).

[^1]: I haven't checked each individually, so I'm not sure yet.